app/services/metadata_api_client/service.rb

```diff
@@ -33,5 +33,11 @@ def self.create(metadata)
     rescue Faraday::UnprocessableEntityError => e
       error_messages(e)
     end
+
+    def self.delete(service_id)
+      connection.delete(ActionController::Base.helpers.sanitize("/services/#{service_id}"))
```

Comment:
To avoid brakeman warning issue: https://app.circleci.com/pipelines/github/ministryofjustice/fb-editor/9460/workflows/bd3c8b97-4e1d-4342-8e49-72f51f87a708/jobs/55729

Reply:
Oh! that's unfortunate. Hmmm, I guess it is ok for now. Ideally service_id should always be enforced/checked to make sure a service by that ID exists and no injection is possible, and if we are sure that is happening, we can add an exception for brakeman.